src/utils/BrowserDetection.ts

```diff
@@ -14,3 +14,47 @@ export const isIE11 = () =>
 
 // Edge 20+
 export const isEdge = () => canUseDOM && !isIE() && 'styleMedia' in window;
+
+export const getChromeVersion = () => {
+  if (canUseDOM) {
+    const match = window.navigator.userAgent.match(/Chrome\/([0-9]+)\./);
+    return match ? parseFloat(match[1]) : false;
```

Comment:
Chromium is used in linux desktop systems.

```suggestion
    const match = window.navigator.userAgent.match(/Chrom(e|ium)\/([0-9]+)\./); 
    return match ? parseFloat(match[2]) : false;
```